prod/westeurope/linux/appbackendl1/app_service/terragrunt.hcl

```diff
@@ -86,8 +86,20 @@ include {
   path = find_in_parent_folders()
 }
 
+locals {
+  # allow application insight web test to call the function.
+  app_insights_ip_west_europe = [
+    "51.144.56.96/28",
+    "51.144.56.112/28",
+    "51.144.56.128/28",
+    "51.144.56.144/28",
+    "51.144.56.160/28",
+    "51.144.56.176/28",
+  ]
+}
+
```

Comment:
what do u think to set it as secret into keyvault? In this way we can set these values only one time

Reply:
> what do u think to set it as secret into keyvault? In this way we can set these values only one time

mmm I don't like the idea of mixing up secrets with configurations. Those list of IPs are public and I think it better to maintain than public in github.
However if the goal is do not duplicate configuration there might be a better approach: like the one showed in this stackoverflow:
https://stackoverflow.com/questions/61140984/terragrunt-and-common-variables

Reply:
Great, I like it :) We have a lot of functions and app services and it could be more easy to manage this values